admin_pages/registrations/templates/reg_admin_details_side_meta_box_registrant.template.php

```diff
@@ -24,7 +24,7 @@
         <span class="admin-side-mbox-label-spn lt-grey-txt float-left">
             <?php esc_html_e('Email', 'event_espresso'); ?>
         </span>
-        <a href="mailto:<?php echo $email; // sanitized ?>">
+        <a href="mailto:<?php echo esc_attr($email); ?>">
```

Comment:
we'll need to confirm that this will work after escaping

Reply:
I tested it again and it works fine but we will ask Mohammad or Sam to check whole things again.

Reply:
awesome, thnx 🙇🏻